inference-engine/ie_bridges/python/src/openvino/test_utils/CMakeLists.txt

```diff
@@ -47,3 +47,13 @@ add_custom_command(TARGET ${TARGET_NAME}
     POST_BUILD
     COMMAND ${CMAKE_COMMAND} -E copy ${PYTHON_BRIDGE_SRC_ROOT}/src/openvino/test_utils/__init__.py ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/__init__.py
 )
+
+file (GLOB SRC
+        ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp
+        )
+
+file (GLOB HEADERS
+        ${CMAKE_CURRENT_SOURCE_DIR}/*.hpp
+        )
+
+add_clang_format_target(${TARGET_NAME}_clang FOR_SOURCES ${SRC} ${HEADERS})
```

Comment:
can we use `FOR_TARGETS` with `TARGET_NAME`?

Reply:
I tried, but target didn't get hpp files

Reply:
Because they are not a part of
```
file(GLOB SOURCE
        ${CMAKE_CURRENT_SOURCE_DIR}/test_utils_api.pyx
        ${CMAKE_CURRENT_SOURCE_DIR}/test_utils_api_impl.cpp)
```
We need to add them to sources as well.

Reply:
updated